src/theme/shadows/index.ts

```diff
@@ -1,5 +1,7 @@
 import { Theme } from "@mui/material/styles";
 
-const Shadows = (theme: Theme) => theme.shadows;
+const Shadows = (theme: Theme) => {
+    return { ...theme.shadows, 1: "rgba(76, 78, 100, 0.22) 0px 2px 10px 0px" };
```

Comment:
Let's give it a custom name instead of reassigning existing one, because it's already used by other components.

Reply:
renamed to `customShadow`, a more descriptive naming convention can be decided on if many more shadows are added